event_project/readme/USAGE.rst

```diff
@@ -0,0 +1,6 @@
+To use this module, you need to:
+
+#. Create a project or duplicate an existing one that fits your needs.
+#. Go to an event or create a brand new one and assign the previous project.
+#. The project will inherit its name and will fit its end date to the begging of
```

Comment:
A new project will be created that will inherit the name of the selected project plus the dates of the event.
The new project will fit its end date to the end date of the event.

Reply:
I've changed the usage description, could you review it?